### DISCUSSION TITLE
hexo+next+waline，为啥部署后打开页面是评论系统？

### DISCUSSION THREAD

**Discussion by ldefault111** [walinejs/waline#1058] [Q&A] [answered]
按照Waline官网的步骤来的，Hexo博客，NexT主题，hexo s显示的没有问题，但是部署后打开网页就是显示图片这样，为什么呀？
![image](https://user-images.githubusercontent.com/103192925/170068987-dfeced02-e4fc-425f-9700-ac72909ecc98.png)

**Comment by lizheming** [comment, marked as answer]
你这个是 Valine Admin 的页面，这个你自行检查下吧，官网文档没说要装 Valine Admin。

**Reply by ldefault111** [reply]
谢谢，大概是之前配的valine在leancloud里没有删干净
